editor/src/components/canvas/controls/insert-mode-control-container.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-expressions */
```

Comment:
What is this being used for? Please can we only use this around the actual expressions we want to ignore.

Reply:
eslint has a problem if you call a function that involves `?.` the no-unused-expressions rule is useful to catch the mistake of a line of code that does nothing, but I wonder if I should just disable this rule, given that we don't usually make that kind of mistake.

alternatively we should bump eslint, but that involves changing the eslint we embed in the editor. or we should have two eslint dependencies, one aliased for bundling, and one for our own actual use.